### PR TITLE
AI Sitegen Capability Check

### PR DESCRIPTION
**Pull request by officiallygod**
Screenshot 2024-01-17 at 12 35 47 PM

### CODE REVIEW

File: src/OnboardingSPA/components/StartOptions/index.js [hunk 1]
```diff
@@ -7,6 +7,8 @@ import { memo } from '@wordpress/element';
 import { store as nfdOnboardingStore } from '../../store';
 
 const StartOptions = ( { questionnaire, oldFlow, options } ) => {
+	const AI_FLOW = 'ai';
+	const ONBOARDING_FLOW = 'onboarding';
```
Comment (arunshenoy99) [inline src/OnboardingSPA/components/StartOptions/index.js:11]: Should we move to https://github.com/newfold-labs/wp-module-onboarding/blob/trunk/src/constants.js?

File: src/OnboardingSPA/components/StartOptions/index.js [hunk 2]
```diff
@@ -53,9 +55,9 @@ const StartOptions = ( { questionnaire, oldFlow, options } ) => {
 	};
 	const selectFlow = ( flow ) => {
 		switch ( flow ) {
-			case 'onboarding':
+			case ONBOARDING_FLOW:
```
Comment (arunshenoy99) [inline src/OnboardingSPA/components/StartOptions/index.js:58]: Should we maintain consistency by changing the terms onboarding, ai and sitegen to align with what we already have, which are, wp-setup, ecommerce, and sitegen.

File: src/OnboardingSPA/components/StartOptions/index.js (hunk 1 above)
Comment (officiallygod) [reply src/OnboardingSPA/components/StartOptions/index.js:11]: Sounds Good, Let me do that.

File: src/OnboardingSPA/components/StartOptions/index.js (hunk 2 above)
Comment (officiallygod) [reply src/OnboardingSPA/components/StartOptions/index.js:58]: I was doubtful as of ONBOARDING_FLOW covers both wp-setup as well as ecommerce, lemme look and make it aligned

File: src/OnboardingSPA/components/ImageUploader/components/ImageUploaderWithText/stylesheet.scss
Comment (officiallygod) [inline src/OnboardingSPA/components/ImageUploader/components/ImageUploaderWithText/stylesheet.scss:1]: This fixes the Pink `X` button on the Logo Step.

**Review by arunshenoy99 — APPROVED** [review]

**Pull request merged**